### PR TITLE
add nano pytorch example cat-vs-dog

### PR DESCRIPTION
**Pull request by Django-Jiang**

### CODE REVIEW

File: python/nano/example/pytorch/pl-cat-vs-dog/pl-cat-vs-dog.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import os
```
Comment (yangw1234) [inline python/nano/example/pytorch/pl-cat-vs-dog/pl-cat-vs-dog.py:1]: You should add the license header.

See other files for reference.

File: python/nano/example/pytorch/nano-cat-vs-dog/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Nano Pytorch cat-vs-dog Example
```
Comment (yangw1234) [inline python/nano/example/pytorch/nano-cat-vs-dog/README.md:1]: let's move this to another standalone directory called `benchmark`

File: python/nano/example/pytorch/nano-cat-vs-dog/nano-cat-vs-dog.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+#
```
Comment (yangw1234) [inline python/nano/example/pytorch/nano-cat-vs-dog/nano-cat-vs-dog.py:1]: I think we only need one `.py` file for performance testing

File: python/nano/example/pytorch/pl-cat-vs-dog/pl-cat-vs-dog.py (hunk 1 above)
Comment (Django-Jiang) [reply python/nano/example/pytorch/pl-cat-vs-dog/pl-cat-vs-dog.py:23]: Added

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**